python/activator/activator.py

```diff
@@ -195,6 +225,105 @@ def create_app():
         sys.exit(3)
 
 
+def dict_to_fanned_out_visit(obj, ctx):
+    """
+    Converts object literal(dict) to a Fanned Out instance.
+    Args:
+        ctx (SerializationContext): Metadata pertaining to the serialization
+            operation.
+        obj (dict): Object literal(dict)
+    """
+
+    if obj is None:
+        return None
+
+    return FannedOutVisit(**obj)
+
+
+def keda_start():
+
+    try:
+        setup_usdf_logger(
+            labels={"instrument": instrument_name},
+        )
+
+        # Initialize local registry
+        registry = LocalRepoTracker.get()
+        registry.init_tracker()
+
+        # Check initialization and abort early
+        _get_consumer()
+        _get_storage_client()
+        _get_central_butler()
+        _get_local_repo()
+
+        _log.info("Worker ready to handle requests.")
+
+    except Exception as e:
+        _log.critical("Failed to start worker; aborting.")
+        _log.exception(e)
```

Comment:
As written, the code will continue trying to execute instead of aborting.
```suggestion
        _log.exception(e)
        sys.exit(1)
```